OpenXMLSDK.Engine/Word/ReportEngine/Renders/BarChartExtensions.cs

```diff
@@ -315,11 +315,7 @@ private static Run CreateBarGraph(BarModel chartModel, OpenXmlPart documentPart)
 
             // Add the Value Axis.
             dc.ValueAxis valAx = plotArea.AppendChild<dc.ValueAxis>(new dc.ValueAxis(new dc.AxisId() { Val = new UInt32Value(48672768u) },
-                new dc.Scaling(new dc.Orientation()
-                {
-                    Val = new DocumentFormat.OpenXml.EnumValue<dc.OrientationValues>(
-                        DocumentFormat.OpenXml.Drawing.Charts.OrientationValues.MinMax)
-                }),
+                chartModel.ValuesAxisScaling.GetScaling(),
```

Comment:
Please check null value for ValuesAxisScaling field

Reply:
If null, keep default value

Reply:
Done